src/components/node/IteratorHelperNode.tsx

```diff
@@ -78,15 +78,15 @@ const IteratorHelperNode = ({ data, selected }: IteratorHelperNodeProps) => {
                     icon={icon}
                     parentNode={parentNode}
                     selected={selected}
-                    type={type}
+                    type={name}
```

Comment:
Could you please rename the `type` prop?

Reply:
Yeah, that's a good idea